client/filestore-client/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "clean": "del coverage_report src/**/*.js src/**/*.map test/**/*.js test/**/*.map",
     "build": "tsc",
-    "test": "npm run clean && nyc mocha"
+    "test": "nyc mocha"
```

Comment:
+1, I think we can remove this part from all test commands and the developer runs `clean` if she needs to...

Reply:
@paolobueno sounds good to me :+1: